<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>laue_overlay_editor</class>
 <widget class="QWidget" name="laue_overlay_editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>565</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>565</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit Overlay</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDoubleSpinBox:disabled {background-color: LightGray;}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QVBoxLayout" name="crystal_editor_layout"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="max_energy_label">
     <property name="text">
      <string>Max Energy:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="min_energy_label">
     <property name="text">
      <string>Min Energy:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ScientificDoubleSpinBox" name="max_energy">
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>8</number>
     </property>
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
     <property name="value">
      <double>35.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="ranges_group_box">
     <property name="title">
      <string>Widths</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QCheckBox" name="enable_widths">
        <property name="text">
         <string>Enable Widths</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="ScientificDoubleSpinBox" name="eta_width">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string>°</string>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="ScientificDoubleSpinBox" name="tth_width">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string>°</string>
        </property>
        <property name="decimals">
         <number>8</number>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="eta_width_label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>η</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="tth_width_label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>2θ</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="width_shape_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Width Shape:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="width_shape">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="ScientificDoubleSpinBox" name="min_energy">
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>8</number>
     </property>
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
     <property name="value">
      <double>5.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScientificDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>scientificspinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>min_energy</tabstop>
  <tabstop>max_energy</tabstop>
  <tabstop>enable_widths</tabstop>
  <tabstop>width_shape</tabstop>
  <tabstop>tth_width</tabstop>
  <tabstop>eta_width</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
